Source/WebKit/Platform/cocoa/CocoaHelpers.mm

```diff
@@ -186,6 +188,120 @@
     });
 }
 
+// MARK: NSDictionary helper methods.
+
+NSDictionary *dictionaryWithLowercaseKeys(NSDictionary *dictionary)
+{
+    if (!dictionary.count)
+        return @{ };
+
+    NSMutableDictionary *newDictionary = [NSMutableDictionary dictionaryWithCapacity:dictionary.count];
+    for (NSString *key in dictionary.allKeys) {
+        if (![key isKindOfClass:NSString.class]) {
+            ASSERT_NOT_REACHED();
+            continue;
+        }
+
+        newDictionary[key.lowercaseString] = dictionary[key];
+    }
+
+    return newDictionary;
+}
+
+NSDictionary *mergeDictionaries(NSDictionary *dictionaryA, NSDictionary *dictionaryB)
+{
+    if (!dictionaryB.count)
+        return dictionaryA;
+
+    if (!dictionaryA.count)
+        return dictionaryB;
+
+    NSMutableDictionary *mergedDictionary = [dictionaryA mutableCopy];
+
+    for (id key in dictionaryB.allKeys) {
+        if (!dictionaryA[key])
+            mergedDictionary[key] = dictionaryB[key];
+    }
+
+    return mergedDictionary;
+}
+
+NSDictionary *mergeAndSetValues(NSDictionary *dictionaryA, NSDictionary *dictionaryB)
+{
+    if (!dictionaryB.count)
+        return dictionaryA;
+
+    if (!dictionaryA.count)
+        return dictionaryB;
+
+    NSMutableDictionary *newDictionary = [dictionaryA mutableCopy];
+
+    for (id key in dictionaryB.allKeys)
+        newDictionary[key] = dictionaryB[key];
+
+    return newDictionary;
+}
+
+// MARK: NSLocale helper methods.
+
+NSSet *localeIdentifiersForMostWidelyUsedLanguages()
```

Comment:
Where is this used?

Reply:
absolutely nowhere. added by accident. will remove